Dockerfile

```diff
@@ -1,9 +1,12 @@
-FROM alpine:3.14
+ARG ALPINE_VERSION=3.14
+FROM alpine:${ALPINE_VERSION}
 LABEL Maintainer="Tim de Pater <code@trafex.nl>"
 LABEL Description="Lightweight container with Nginx 1.20 & PHP 8.0 based on Alpine Linux."
+# Setup document root
+WORKDIR /var/www/html
```

Comment:
Why did you move this to the top?

Reply:
By caching the layers and to avoid the mkdir of line 42.

Reply:
Good improvement :+1: